pkg/instance/instance.go

```diff
@@ -69,6 +69,22 @@ type ObsyConfig struct {
 	prometheusRemoteWriteExporterEndpoint string
 }
 
+// TsharkCollectorConfig represents the configuration for the tshark collector
+type TsharkCollectorConfig struct {
+	// volumeSize is the size of the volume to use for the tshark collector
+	volumeSize string
```

Comment:
Is it possible to change this type to resource.Quantity ? like the one that comes out of  this func: https://pkg.go.dev/k8s.io/apimachinery@v0.28.2/pkg/api/resource#ParseQuantity

then it will be less prone to errors